dockstore-integration-testing/src/test/java/io/dockstore/client/cli/Ga4GhTRSAPIWorkflowIT.java

```diff
@@ -510,54 +513,28 @@ void testDifferentCaseForZipDownload() throws ApiException, URISyntaxException,
 
         workflowApi.refresh(workflowByPathGithub.getId(), false);
 
-        // test parameter with wildcard header, should get zip
-        byte[] arbitraryURL = CommonTestUtilities.getArbitraryURL(
-            "/ga4gh/trs/v2/tools/" + URLEncoder.encode("#workflow/" + DOCKSTORE_TEST_USER2_RELATIVE_IMPORTS_WORKFLOW, StandardCharsets.UTF_8) + "/versions/master"
-                + "/" + DescriptorTypeWithPlain.CWL
-                + "/files?format=zip", new GenericType<>() {
-                }, webClient, MediaType.MEDIA_TYPE_WILDCARD);
-        checkOnZipFile(arbitraryURL, DescriptorLanguage.CWL);
-
-        // even more wildcard, should get zip
-        arbitraryURL = CommonTestUtilities.getArbitraryURL(
-            "/ga4gh/trs/v2/tools/" + URLEncoder.encode("#workflow/" + DOCKSTORE_TEST_USER2_RELATIVE_IMPORTS_WORKFLOW, StandardCharsets.UTF_8) + "/versions/master"
-                + "/" + DescriptorTypeWithPlain.CWL
-                + "/files?format=zip", new GenericType<>() {
-                }, webClient, MediaType.WILDCARD);
-        checkOnZipFile(arbitraryURL, DescriptorLanguage.CWL);
-
-        // json header with zip format, should get error
-        arbitraryURL = CommonTestUtilities.getArbitraryURL(
```

Comment:
This test appears to be a duplicate of the test directly above it, so it was deleted.